programs/client/Client.cpp

```diff
@@ -234,6 +240,35 @@ try
     MainThreadStatus::getInstance();
     setupSignalHandler();
 
+#if USE_SSL
+    class ClientCertificateHandler: public Poco::Net::InvalidCertificateHandler
+    {
+    private:
+        bool ignore = false;
+    public:
+        explicit ClientCertificateHandler(bool ignore_): Poco::Net::InvalidCertificateHandler(false), ignore(ignore_) {}
+        void onInvalidCertificate(const void* /* pSender*/, Poco::Net::VerificationErrorArgs& error) override
+        {
+            const Poco::Net::X509Certificate& cert = error.certificate();
+            std::cout << "\n";
+            std::cout << (ignore ? "WARNING" : "ERROR") << ": Certificate verification failed\n";
```

Comment:
Maybe add these warnings to the list of warning from server be consistent? And also not to print this warning in case when `no-warnings` is specified?

Reply:
this warning is completely client-side... I'm not even sure it should be affected by `no-warnings` since it's for server side warnings...